rxandroidble/src/main/java/com/polidea/rxandroidble/internal/operations/CharacteristicLongWriteOperation.java

```diff
@@ -209,4 +214,58 @@ public Boolean call(Object emission) {
             }
         };
     }
+
+    private static Func1<Observable<? extends Throwable>, Observable<?>> retryOperationStrategy(
```

Comment:
Maybe `errorIsRetriableAndAccordingTo(...)`? Then we could have `.retryWhen(errorIsRetriableAndAccordingTo(writeOperationRetryStrategy,...))`

Reply:
Ok.